src/components/NavBar/NavItem/index.tsx

```diff
@@ -14,10 +23,30 @@ function NavItem({ onClick, label, isActive }: Props) {
   };
   return (
     <div title="item" onClick={activation} role="alert">
-      {label && (
-        <p className={`${isActive ? styles.active : ''}${styles.label}`}>
-          {label}
-        </p>
+
+      {items?.submenu ? (
+        <>
+          <p
+            className={`${isActive ? styles.active : ''}${styles.label}`}
+            aria-haspopup="menu"
+            onClick={() => setDropdown((prevState:boolean) => !prevState)}
+          >
+            {items.name}
+            {
+            dropdown
+              ? <span><FontAwesomeIcon icon={faChevronUp} /></span>
+              : <span><FontAwesomeIcon icon={faChevronDown} /></span>
+}
+          </p>
+          <SubDrop
+            submenus={items.submenu}
+            dropdown={dropdown}
+          />
+        </>
+      ) : label && (
+      <p className={`${isActive ? styles.active : ''}${styles.label}`}>
```

Comment:
Does this work properly? Because it seems that if isActive is true the full label ends up being `.active.label` all together and it may create conflicts

Reply:
done